lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralLinearModelAlgorithm.cxx

```diff
@@ -766,8 +818,26 @@ OptimizationAlgorithm GeneralLinearModelAlgorithm::getOptimizationAlgorithm() co
   return solver_;
 }
 
+// If solver is MultiStart, check the bounds contain all existing starting points
+void GeneralLinearModelAlgorithm::checkMultiStartConsistentWithOptimizationBounds(const OptimizationAlgorithm & solver, const Interval & optimizationBounds) const
```

Comment:
Shouldn't this test be done in MultiStart? When you set the problem in MultiStart, you can check if it has bounds and see if they are compatible with the sample, and when you set the sample you can check if it is compatible with the bounds, if any.

Reply:
Yes, you are probably right. Of course, this means that a possible inconsistency will only be detected in the `run()` method, but this is a price I am willing to pay.

Reply:
You are sweet but are the others willing to pay this price? Not me. You better understand why I limited my changes to the run() method now?

Reply:
I do not understand what you mean. Do you want to keep this test or is enough to hand it over to MultiStart? If we hand it over to MultiStart, then we need to accept that the test will only be performed at the beginning of `GLMAlgo::run` because the optimization problem is only defined there.

Reply:
Anyway, #1797 has been merged, so MultiStart already implements this check. Removing it from GLMAlgo.